src/Http/Http.Extensions/src/UriHelper.cs

```diff
@@ -171,16 +171,16 @@ public static string Encode(Uri uri)
         /// and other HTTP operations.
         /// </summary>
         /// <param name="request">The request to assemble the uri pieces from.</param>
-        /// <returns></returns>
+        /// <returns>The encoded string version of the url from <paramref name="request"/>.</returns>
         public static string GetEncodedUrl(this HttpRequest request)
         {
             return BuildAbsolute(request.Scheme, request.Host, request.PathBase, request.Path, request.QueryString);
         }
         /// <summary>
-        /// Returns the relative url 
+        /// Returns the relative url
         /// </summary>
         /// <param name="request">The request to assemble the uri pieces from.</param>
```

Comment:
Everywhere that `uri`, `Uri`, `url`, `Url` appear in "prose", they must be all lowercase. Only when it's referencing an identifier (property, type name, etc.), it should be PascalCase.

Reply:
I assume you mean uppercase, not lowercase?

Reply:
Sorry yes!